app/controllers/accounts_controller.rb

```diff
@@ -3,11 +3,11 @@ class AccountsController < ApplicationController
   customer_tab  :all
   before_action :authenticate_user!
   before_action :check_acting_as
-  before_action :init_account, only: [:show, :user_search, :transactions, :transactions_in_review, :suspend, :unsuspend]
+  before_action :init_account, only: [:show, :user_search, :transactions, :suspend, :unsuspend]
```

Comment:
We don't need to do anything here, but the rubocop/codeclimate warning is interesting about explicitly defining actions in the controller when using the `before_action`. Just something to keep in mind going forward.